MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -61,4 +61,62 @@ class VisionManagerTests: XCTestCase {
             "VisionManager should be successfully deallocated after calling destroy and releasing the object"
         )
     }
+
+    func testPerformanceConfigMergedTranslatesToSamePerformance() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance = ModelPerformance(mode: .dynamic, rate: .high)
+        visionManager.modelPerformanceConfig = .merged(performance: performance)
+
+        // Then
+        XCTAssertEqual(visionManager.modelPerformance, performance)
+    }
+
+    func testPerformanceConfigSeparateDynamicModeTranslatesToHigherRate() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance1 = ModelPerformance(mode: .dynamic, rate: .low)
```

Comment:
Here and below:

Same as above, we can move initialization for `performance1`/`performance2` into `Given` section.